MPhys/Single_Emitter/single_emitter_test_new_int1000.py

```diff
@@ -0,0 +1,353 @@
+#!/usr/bin/env python
```

Comment:
This file is probably unneccessary - it was useful at one point to be able to run one example quickly, but it's a repeat from within a larger script.